### DISCUSSION TITLE
I made Light-Dark mode icons

### DISCUSSION THREAD

**Discussion by Human79** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#56] [General]
I made Light-Dark mode icons. I will keep perfecting them in the future.
Attaching icons. Please give me credit.
 Regards.
[WIFI Icons.zip](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/files/6644498/WIFI.Icons.zip)

**Comment by chris1111** [comment]
Sorry but I have made a better then your long time ago ; then that's why I made a bue Icon

**Comment by Tetoalb** [comment]
Can I upgrade my MacBook to the latest? Would it work? Thanks

Sent from my iPhone

On Jun 13, 2021, at 4:00 PM, chris1111 ***@***.***> wrote:

﻿

Sorry but I have made a better then your long time ago ; then that's why I made a bue Icon
